formlayout.py

```diff
@@ -265,6 +265,39 @@ def text(self):
         return self.lineedit.text()
 
 
+class SliderLayout(QHBoxLayout):
+    """QSlider with QLabel"""
+    def __init__(self, value, parent=None):
+        QHBoxLayout.__init__(self)
+        index = value.find('@')
+        if index != -1:
+            value, default = value[:index], int(value[index+1:])
+        else:
+            default = False
+        parsed = value.split(':')
+        self.slider = QSlider(Qt.Horizontal)
+        if parsed[-1] == '':
+            self.slider.setTickPosition(2)
+            parsed.pop(-1)
+        if len(parsed) == 2:
+            self.slider.setMaximum(int(parsed[1]))
+        elif len(parsed) == 3:
+            self.slider.setMinimum(int(parsed[1]))
+            self.slider.setMaximum(int(parsed[2]))
+        if default:
+            self.slider.setValue(default)  # always set value in last
+        self.slider.valueChanged.connect(self.update)
```

Comment:
**PyQt API 1 compatibility**: we need to handle the compatibility with old-style signals and slots. In this case, I'm expecting something like:
```python
        if SIGNAL is None:
            self.slider.valueChanged.connect(self.update)
        else:
            self.connect(self.slider, SIGNAL("valueChanged(int)"), self.update)  # need to be tested!
```